ts/utils/convertDateToWordDistance.ts

```diff
@@ -7,26 +7,23 @@ import { format, formatDateAsLocal } from "./dates";
  * This function converts the distance from now to date in :
  * H.mm, yesterday, MM/DD (or DD/MM) and MM/DD/YY (or DD/MM/YY) depending on the system locale
  */
+
 export function convertDateToWordDistance(
   date: Date,
   lastDayLabel: string,
-  todayAtLabel?: string,
-  invalidDateLabel?: string
+  todayAtLabel?: string
 ): string {
   const distance = differenceInCalendarDays(new Date(), date);
   // 0 days, distance < one day
   if (distance < 1) {
-    const formatted = format(date, "H.mm");
+    const formatted = format(date, "H:mm");
     return todayAtLabel ? `${todayAtLabel} ${formatted}` : formatted;
   } // distance = 1 day
   else if (distance === 1) {
     return lastDayLabel;
   } // 1 day < distance < 365 days, current year
   else if (distance > 1 && distance < 365) {
     return formatDateAsLocal(date);
-  } // the original date is invalid (distance is NaN)
-  else if (isNaN(distance)) {
```

Comment:
Why did you chose to remove this path? 
I think it is a safe check.

Type declaration may help us only at compile time, but not at run time.
This is what happens if we remove the check and for any reason from the backend we got an invalid data where we expected a `Date`:
<img src="https://user-images.githubusercontent.com/3959405/100325843-70a92f80-2fc9-11eb-8806-973bac9fd393.png" height="550"/>

Instead keeping it as it was we may use it as it follows:
```typescript
    const uiDate = convertDateToWordDistance(
      message.created_at,
      I18n.t("messages.yesterday"),
      I18n.t("messages.today"),
      "Data non riconosciuta"
    );
```
and the result would be more clear:
<img src="https://user-images.githubusercontent.com/3959405/100326256-0349ce80-2fca-11eb-9a75-2ac140c4eab9.png" height="550"/>

Reply:
We discussed that with @Undermaken and confirmed that messages never reach the app's state in case even a single `created_at` is malformed, how did you manage to obtain this result?

Reply:
This is true.
Btw I changed my mind and I don't see any fault to leave this safety test.
It could be restored. If the API change this can prevent errors